### PR TITLE
[DOC-457] Added known issues to Manual review guideline

### PR DESCRIPTION
**Pull request by aishwarya24**
https://deploy-preview-23634--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-voyager/known-issues/postgresql/

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/known-issues/postgresql.md [hunk 1]
```diff
@@ -170,3 +187,727 @@ Suggested workaround is to change the INET column to TEXT for the index creation
...
+$$ LANGUAGE plpgsql;
+
+CREATE TRIGGER check_no_time_overlap_trigger
+    BEFORE INSERT OR UPDATE ON meeting
+    FOR EACH ROW
+    EXECUTE FUNCTION check_no_time_overlap();
+```
+
+<!-- The following GiST index is also required for better performance of constraints, because the source constraint uses gist method but the target YugabyteDB doesn't support GIST indexes. need to do some workaround for that as well [refer](LINK DOCS).. -->
+
```
Comment (priyanshi-yb) [inline docs/content/preview/yugabyte-voyager/known-issues/postgresql.md:490]: remove this para not required

File: docs/content/preview/yugabyte-voyager/known-issues/postgresql.md [hunk 2]
```diff
@@ -170,3 +187,727 @@ Suggested workaround is to change the INET column to TEXT for the index creation
...
+**GitHub**: [Issue #9698](https://github.com/yugabyte/yugabyte-db/issues/9698)
+
+**Description**: If you have indexes on some complex types such as TSQUERY, TSVECTOR, JSON, UDTs, citext, and sp on, those will error out in import schema phase to target YugabyteDB with the following error:
+
+```output
+ ERROR:  INDEX on column of type '<TYPE_NAME>' not yet supported
+```
+
+**Workaround**: Currently, there is no workaround, but you can cast these data types to supported types, which may require adjustments on the application side to query the column using the index. Ensure you address these changes before modifying the schema.
```
Comment (priyanshi-yb) [inline docs/content/preview/yugabyte-voyager/known-issues/postgresql.md:601]: Currently, there is no workaround, but you can cast these data types in the index definition to supported types, which may require adjustments on the application side while querying the column using the index. Ensure you address these changes before modifying the schema

File: docs/content/preview/yugabyte-voyager/known-issues/postgresql.md [hunk 3]
```diff
@@ -170,3 +187,727 @@ Suggested workaround is to change the INET column to TEXT for the index creation
...
+    ON public.test_gin_json USING gin (text, text1);
+```
+
+---
+
+### Policies on users in source require manual user creation
+
+**GitHub**: [Issue #1655](https://github.com/yugabyte/yb-voyager/issues/1655)
+
+**Description**: If there are policies in the source schema for users in database, the users have to be created manually on target YugabyteDB, as currently migration USER/GRANT is not supported. Skipping the manual user creation will return an error in import schema as follows:
```
Comment (priyanshi-yb) [inline docs/content/preview/yugabyte-voyager/known-issues/postgresql.md:809]: If there are policies in the source schema for users in the database, the USERs have to be created manually on target YugabyteDB, as currently the migration of USER/GRANT is not supported. Skipping the manual user creation will return an error in import schema as follows:

File: docs/content/preview/yugabyte-voyager/known-issues/postgresql.md [hunk 4]
```diff
@@ -170,3 +187,727 @@ Suggested workaround is to change the INET column to TEXT for the index creation
...
+
+**Workaround**: To implement exclude constraints, follow this workaround:
+
+1. Create a trigger: Set up a TRIGGER for INSERT or UPDATE operations on the table. This trigger will use the specified expression to search the relevant columns for any potential violations.
+
+1. Add indexes: It is crucial to create an INDEX on the columns involved in the expression. This helps ensure that the search operation performed by the trigger does not negatively impact performance.
+
+Note that creating an index on the relevant columns is essential for maintaining performance. Without an index, the trigger's search operation can degrade performance.
+
+**Caveats** : Note that there are specific issues related to [creating indexes on certain data types](LINK to other issue) in YugabyteDB. Depending on the data types involved, additional workarounds may be required to ensure optimal performance for these constraints.
```
[the remark this answers is not in the document]
Comment (priyanshi-yb) [reply docs/content/preview/yugabyte-voyager/known-issues/postgresql.md]: @aishwarya24 can we please also update this part `on certain data types or certain index methods in YugabyteDB. Depending on the data types or method involved,` to include the certain index methods as well.

**Review by priyanshi-yb — APPROVED** [review]

**Review by ddhodge — APPROVED** [review]

**Pull request merged**